server/dump.go

```diff
@@ -44,7 +43,11 @@ func (p *Plugin) createDump(ctx context.Context, min, max time.Time, remoteStora
 		}
 	}
 
-	db, err := tsdb.Open(dumpDir, log.NewNopLogger(), nil, tsdb.DefaultOptions(), nil)
+	db, err := tsdb.Open(dumpDir, p.logger, nil, &tsdb.Options{
+		MinBlockDuration:           int64(2 * time.Hour / time.Millisecond),
+		MaxBlockDuration:           int64(6 * time.Hour / time.Millisecond),
```

Comment:
How did we come up with this number? Would be good to have some reasoning explained in a comment since it's deviating form the default.

Reply:
The longer the block the better it should have the compaction. Added the comment.